### PR TITLE
[DEV-1067 DEV-1072] webinar detail page

### PR DESCRIPTION
**Pull request by MarcoPonchia**
#### List of Changes

Add webinar detail page

#### Motivation and Context

Add webinar detail page and  routing

#### How Has This Been Tested?

User can see webinar detail page on: http://localhost:3000/webinars/comunicazioni-a-valore-legale

#### Screenshots (if appropriate):
screenshot-localhost_3000-2023 11 14-12_11_05

#### Types of changes

- [ ] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/nextjs-website/src/app/webinars/[webinarSlug]/page.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+}
+
+const Page = async ({ params }: { params: Params }) => {
+  const webinar = await getWebinar(params?.webinarSlug);
+  const { webinar: webinarLabels } = translations;
+
+  return (
+    <>
+      {webinar.startInfo && (
+        <StartInfo
```
Comment (marcobottaro) [inline apps/nextjs-website/src/app/webinars/[webinarSlug]/page.tsx:24]: The background color of this section is not #FAFAFA, but #F2F2F2 and the title is too close to the cards.
image

**Review by marcobottaro — APPROVED** [review]

**Comment by marcobottaro** [comment]
image

vertical padding should be 64px

**Comment by marcobottaro** [comment]
image
`font-weight: 600; font-size: 24px;`

**Review by marcobottaro — CHANGES_REQUESTED** [review]
Fix these little differences with Figma:
- https://github.com/pagopa/developer-portal/pull/363#issuecomment-1810050595
- https://github.com/pagopa/developer-portal/pull/363#issuecomment-1810053344
- https://github.com/pagopa/developer-portal/pull/363#issuecomment-1810068129

**Review by marcobottaro — APPROVED** [review]

**Review by kin0992 — APPROVED** [review]

**Pull request merged**